tests/unit/test_filterout_utils.py

```diff
@@ -129,4 +130,87 @@ def test_filter_out():
         'authors.full_name',
         'report_numbers'
     ]
-    assert len(filter_out(conflicts, fiels)) is 4
+    assert len(filter_conflicts(conflicts, fiels)) is 4
```

Comment:
there's a typo (carried over by autocomplete I guess), s/fiels/fields/

Maybe we should add the test for `authors.full`, see if that removes `authors.full_name`.